### PR TITLE
Code for the Youtube Alternatives section

### PR DESCRIPTION
**Pull request by ADepic**
## Description

Resolves: #693

I'm not a programmer, but I know basic html so I gave it a shot. There might be errors so feel free to fix them.

The logos are not 120px * 120px, but I don't have the time to go and find 120 px logos / edit them into 120px using gimp.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -127,6 +127,7 @@
                 <a class="dropdown-item" href="{{prefix}}#encrypt">File Encryption</a>
                 <a class="dropdown-item" href="{{prefix}}#darknets">Self-contained Networks</a>
                 <a class="dropdown-item" href="{{prefix}}#social">Social Networks</a>
+		<a class="dropdown-item" href="{{prefix}}#streaming">Youtube Alternatives</a>
```
Comment (Shifterovich) [inline _layouts/default.html]: Fix indentation

File: assets/img/tools/newpipe.svg [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Shifterovich) [inline assets/img/tools/newpipe.svg]: I'm not sure if we allow svg's. @Vincevrp? Also, perhaps this file should be minified?

**Comment by asddsaz** [comment]
Check out PR #671

**Comment by Kcchouette** [comment]
Duplicate to #671 for me
NewPipe is not an alternative to YouTube, as it uses youtube in backend

**Comment by ADepic** [comment]
Newpipe is one of the most polished youtube apps there is. The danger is, if someone thinks invidious is the best alternative to youtube, since the interface isn't that good, they may just continue to use the youtube app. Newpipe has many features that even the youtube app doesn't, so it would be a lot more appealing. A user is losing out on a lot of convenience for no reason by using invidious on their phone.

**Comment by ADepic** [comment]
I have changed the pull request to contain a mix of youtube frontends and streaming services to youtube frontends only. That way PR #671 can focus on alternative streaming services only.

File: _layouts/default.html [hunk 3]
```diff
@@ -127,6 +127,7 @@
                 <a class="dropdown-item" href="{{prefix}}#encrypt">File Encryption</a>
                 <a class="dropdown-item" href="{{prefix}}#darknets">Self-contained Networks</a>
                 <a class="dropdown-item" href="{{prefix}}#social">Social Networks</a>
+		<a class="dropdown-item" href="{{prefix}}#streaming">Youtube Frontends</a>
```
Comment (Mikaela) [inline _layouts/default.html]: I think this should align with the other a tags.

**Pull request closed without merging**